### DISCUSSION TITLE
[委托] 解救简大姐

### DISCUSSION THREAD

**Discussion by MengYue-MK2000** [share121/inter-knot#4803] [绝区零]
![伪装简](https://github.com/user-attachments/assets/8bb06ff3-75a7-4f50-a60b-fd01e66bf712)
急急急，简大姐被治安局给带走了QAQ，现急需精英绳匠协助解救她；
根据在治安局内应的情报，她目前正在前往雅努斯分局，而且只有1个新人治安官负责押送；
难度非常简单，报酬绝对不会让你失望

**Comment by firyln** [comment]
_(•̀ω•́ 」∠)_

**Comment by The-Place-Where-Winter-Sieeps** [comment]
我去，这么勇。不知道治安局在这有官号吗?小心被逮
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by MengYue-MK2000** [reply]
放心啦，简大姐说了，她在治安局有很多朋友，没问题的啦(●′ω`●)

**Comment by wlmask** [comment]
哎嘿|･ω･｀)这么明目张胆嘛

**Comment by lmingzhe86** [comment]
这个委托我接了！

**Reply by MengYue-MK2000** [reply]
真的吗，好耶(*^▽^*)，我把简大姐手机定位发您，就拜托您了

**Comment by lllliufeng** [comment]
接接接，为什么不接

**Reply by MengYue-MK2000** [reply]
好耶，我把简大姐的定位也发您，报酬不会让您失望的ヾ(^▽^)ノ

**Comment by SWYH9527** [comment]
我不接，我打不过！

**Comment by MengYue-MK2000** [comment]
没事(●′ω`●)我和2位精英绳匠打一个新人治安官，3对1，等救出简大姐就是四打一，优势在我们啦(*^▽^*)，会赢么，会赢的(ง •̀_•́)ง‼

**Comment by HollowInvestigativeAssociation** [comment]
需要援助吗，HIA随时可以为您服务

**Reply by Cypher2024** [reply]
坏啦，协会也有高仿号了

**Comment by WuxuRiver** [comment]
不是，HIA算是官方的人吧

**Comment by XinRenShenJiangZiShu** [comment]
我无法帮助你们现实进入，当可以从精神和网络协助你们，你们应该黑不进治安局的系统

**Comment by JT9364** [comment]
嗯～这种有可能回不来的活Faily就不要接了吧

**Comment by Minapocalis** [comment]
谢瑶，感谢你对我的信任，但臣妾还是那句话臣妾做不到啊！！！

**Comment by ZoneZero14168** [comment]
我相信简姐姐一定可以自己出来的！

**Comment by qinzhibai** [comment]
太可怕了

**Comment by yanER582136479** [comment]
已经好几个月了，听说简大姐居然是治安局的人不知道真假，话说楼主活着出来了吗？
